src/models/Map.js

```diff
@@ -232,10 +232,10 @@ export default class Map{
     this.layerGoogle = new GoogleLayer(
       'http://{s}.google.com/vt/lyrs=s,h&x={x}&y={y}&z={z}',{
         maxZoom: this.maxZoom,
-        //attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
-        //'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+        attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors ',
+        // 'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
         subdomains:['mt0','mt1','mt2','mt3'],
-        zIndex: 2,
+        zIndex: 0,
```

Comment:
@adriana-carmo Please change this to:
© Google  (link to HTTP://map.google.com)
© Greenstand. (link to HTTP://greenstand.org) 
Like this: 
https://docs.mapbox.com/help/getting-started/web-apps/
![image](https://user-images.githubusercontent.com/5744708/123016801-f5441b00-d3fd-11eb-8513-29afb6149f69.png)